mindsworth/My-Diary-Frontend
# #162377815 User should view homepage

#### What does this PR do?
React setup

#### Description of Task to be completed?
- user should land on home page

#### How should this be manually tested?
1. After cloning or pulling the branch to your local
2. Run yarn install to install all necessary packages and run yarn start:dev start your app on port 8080
3. visit the right URL e.g http://localhost:8080 to see the home page.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
-[ #162377815](https://www.pivotaltracker.com/story/show/162377815)

#### Screenshots (if appropriate)
<img width="1680" alt="screenshot 2018-12-03 at 12 45 04 pm" src="https://user-images.githubusercontent.com/14542449/49371925-4f264d80-f6f9-11e8-98ca-9f241ae86416.png">


#### Questions:
N/A
